test/models/lookup_key_test.rb

```diff
@@ -420,4 +420,30 @@ def setup
       assert_valid lookup_key
     end
   end
+
+  test "can update lookup key with default_value without audit error" do
+    loc1 = taxonomies(:location1)
+    loc2 = taxonomies(:location2)
+    location_list = [loc1, loc2]
```

Comment:
nit: can we get rid of the variables?
```
location_list = %w(taxonomies(:location1) taxonomies(:location2))
```

Reply:
Done! Modified to `[taxonomies(:location1), taxonomies(:location2)]` as array of objects expected here.